snuba/migrations/migrate.py

```diff
@@ -38,14 +38,11 @@ def run(dataset: Dataset) -> None:
     schemas: MutableSequence[Schema] = []
     writable_storage = dataset.get_writable_storage()
     if writable_storage:
-        writer = writable_storage.get_table_writer()
-        schemas.append(writer.get_schema())
+        schemas.append(writable_storage.get_table_writer().get_schema())
```

Comment:
Do we even need this now ? Or is it in order to migrate the write schemas before the read ones? If it is the case could you please add a comment that explains it ?

Reply:
I assume it's to enforce the order of migrations, added a comment